causal_testing/testing/causal_test_outcome.py

```diff
@@ -35,22 +43,22 @@ class Positive(CausalTestOutcome):
     """An extension of TestOutcome representing that the expected causal effect should be positive."""
 
     def apply(self, res: CausalTestResult) -> bool:
-        # TODO: confidence intervals?
         if res.test_value.type == "ate":
             return res.test_value.value > 0
-        elif res.test_value.type == "risk_ratio":
+        if res.test_value.type == "risk_ratio":
             return res.test_value.value > 1
+        raise ValueError(f"Test Value type {res.test_value.type} is not valid for this TestOutcome")
 
 
 class Negative(CausalTestOutcome):
     """An extension of TestOutcome representing that the expected causal effect should be negative."""
 
     def apply(self, res: CausalTestResult) -> bool:
-        # TODO: confidence intervals?
         if res.test_value.type == "ate":
             return res.test_value.value < 0
-        elif res.test_value.type == "risk_ratio":
+        if res.test_value.type == "risk_ratio":
             return res.test_value.value < 1
+        raise ValueError(f"Test Value type {res.test_value.type} is not valid for this TestOutcome")
```

Comment:
One can test that correct errors are thrown in the right circumstance.

Reply:
Sounds worth doing 👍. I'll save it for another PR to keep this just linting